dissect/target/plugins/browsers/iexplore.py

```diff
@@ -71,14 +96,33 @@ def __init__(self, target: Target):
                 self.users_dirs.append((user_details.user, cdir))
 
     def check_compatible(self) -> bool:
+        """Perform a compatibility check with the target.
+        This function checks if any of the supported browser directories
+        exists. Otherwise it should raise an ``UnsupportedPluginError``.
+        Raises:
+            UnsupportedPluginError: If the plugin could not be loaded.
+        """
         if not len(self.users_dirs):
             raise UnsupportedPluginError("No Internet Explorer directories found")
 
-    @export(record=IEBrowserHistoryRecord)
-    def history(self) -> Iterator[IEBrowserHistoryRecord]:
+    def open_cache(self, cdir) -> WebCache:
```

Comment:
Probably better to create a `_iter_cache` function, similar to the `_iter_db` function of Firefox/Chrome.